src/main/java/com/google/cloud/teleport/bigtable/ParquetToBigtable.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.bigtable;
+
+import static com.google.cloud.teleport.bigtable.AvroToBigtable.toByteString;
+
+import com.google.bigtable.v2.Mutation;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.ByteString;
+import java.nio.ByteBuffer;
+import java.util.List;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.io.gcp.bigtable.BigtableIO;
+import org.apache.beam.sdk.io.parquet.ParquetIO;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.options.ValueProvider;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.KV;
+
+/**
+ * Dataflow pipeline that imports data from Parquet files in GCS to a Cloud Bigtable table. The
+ * Cloud Bigtable table must be created before running the pipeline and must have a compatible table
+ * schema. For example, if {@link BigtableCell} from the Parquet files has a 'family' of "f1", the
+ * Bigtable table should have a column family of "f1".
+ */
```

Comment:
Can you make this similar to the other pipeline's javadoc with instructions to compile and execute?

Reply:
@sabhyankar Its similar to the one in AvroToBigtable. And none of the other Legacy templates have compile and execute details.

Reply:
Example:
https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/master/src/main/java/com/google/cloud/teleport/templates/BulkCompressor.java